patches/command-line-api/0001-Exclude-dotnet-suggest-from-source-build.patch

```diff
@@ -0,0 +1,215 @@
+From eeee907f22d384322975644dae2c1c1e884ea61d Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Fri, 18 Sep 2020 16:09:49 -0500
+Subject: [PATCH 1/2] Exclude dotnet-suggest from source-build.
```

Comment:
Why not build it?

Reply:
dotnet-suggest isn't used downstream by anything and adds even more prebuilts, so I figured we can skip it for now.

Reply:
Would be nice to include this in the patch desc (vs. looking at `blame` and navigating to this PR).